modules/local/get_software_versions.nf

```diff
@@ -1,18 +1,11 @@
-// Import generic module functions
-include { saveFiles } from './functions'
-
-params.options = [:]
-
 process GET_SOFTWARE_VERSIONS {
```

Comment:
This process is not needed any more is it? It was replaced with another script no?

Reply:
Yes. It will be removed.